hazelcast-client-new/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -132,6 +133,7 @@ protected Document parse(InputStream inputStream) throws Exception {
         DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
         dbf.setNamespaceAware(true);
         final DocumentBuilder builder = dbf.newDocumentBuilder();
+        Locale.setDefault(Locale.getDefault());
```

Comment:
I'm not sure I understand what's done here. but it looks like doing `this.x = this.x`. It's setting default locale to its current value again.